swayidle.1.scd

```diff
@@ -11,7 +11,7 @@ swayidle - Idle manager for Wayland
 # OPTIONS
 
 *-C* <path>
-	The config file to use. By default, the following paths are checked in the following order: $XDG_CONFIG_HOME/swayidle/config, $HOME/.swayidle/config.
+	The config file to use. By default, the following paths are checked in the following order: $XDG_CONFIG_HOME/swayidle/config, $HOME/.config/swayidle/config, $HOME/.swayidle/config.
```

Comment:
`$HOME/.config` is not a unique path being searched, but is the default value for `XDG_CONFIG_HOME` when the environment variable is not explicitly set.

Reply:
Yeah, I wasn't sure how technical to get in the man page about precisely how this works, that in the absence of the env var, a default is injected into the top of the list... seems a bit wordy and unnecessary. I do think it's helpful to indicate that `$HOME/.config` is the default value though. I mean I know that, but (a) many might not, and (b) I wasn't sure swayidle was doing it (due to other config file issues) until I read the code.

I'll try to clean this wording up a bit

Reply:
The default value of `XDG_CONFIG_DIR` is part of the xdg base dir spec, so you don't have to mention anything - it's a bug if an application does not have that default. The original was fine.

Reply:
I know that, and I was still confused enough to motivate me to submit a PR. It's a small convenience found in many other man pages (see dmesg, top, wofi, wayfire for examples). What's wrong with it here?

Reply:
We don't describe this in our other manpages, and we prefer to not document external stuff which is better described elsewhere. For example, we don't document how POSIX shell command line parsing works for the event commands, how file permissions and ownership work, how environment variables function, etc.

When things are unclear, it is better to reference information than duplicate it. This helps ensure that it is understood to be a specification behavior, rather than a random app implementation detail. But that's a separate discussion that would involve PRs to more projects, so let's keep that out.